### PR TITLE
RAC5852: Common-deploy-infrasim: implement a shell script to deploy infrasim

### PR DESCRIPTION
**Pull request by PengTian0**
[RAC-5852](https://rackhd.atlassian.net/browse/RAC-5852)

- Implement a shell script to deploy infrasim. The script should implement methods: deploy, cleanUp, fetchLog

- Implement a groovy script to call the shell script to deploy virtual nodes

### CODE REVIEW

**Review by changev — APPROVED** [review]
+1 

this is important for next co-works!

**Comment by PengTian0** [comment]
test this please

**Comment by PengTian0** [comment]
test this please

File: deployment/deploy_vnodes.sh [hunk 1]
```diff
@@ -0,0 +1,171 @@
+#!/bin/bash -ex
+Usage(){
+    echo "to do"
```
Comment (anhou) [inline deployment/deploy_vnodes.sh]: echo 'to do‘ ？

Comment (PengTian0) [reply deployment/deploy_vnodes.sh]: Thanks.
Fixed.

**Review by anhou — APPROVED** [review]

**Pull request merged**